pocs/core.py

```diff
@@ -427,7 +425,7 @@ def _check_environment(self):
             POCS      Base directory for POCS
         """
         if sys.version_info[:2] < (3, 0):  # pragma: no cover
-            warn("POCS requires Python 3.x to run")
+            warnings.warn("POCS requires Python 3.x to run")
```

Comment:
Why warnings instead of self.logger?

Reply:
It checks the environment before it adds the logger.